### ISSUE TITLE
Authentication to sign up user.. Graphql-js/6-authentication/

### ISSUE THREAD

**Issue by Muppet-training** [howtographql/howtographql#712]
How do I get past this point..?
There is no instructions on how to get an access token for the playground & Prisma deploy 
doesn't have the schema available to sign up a user after re deploying the code 5 times.. I have followed the tutorial word for word & now I cannot continue due to this missing information.
![authentication](https://user-images.githubusercontent.com/10666103/42617668-d6ed0238-85f5-11e8-8c90-b98b38908e3f.png)

**Comment by guledali** [comment]
I got stuck on that part too take a screenshot on your playground when you have send your mutation!

Thats the thing when you create your mutation you should see field called token
![inkedmutation-token](https://user-images.githubusercontent.com/20647282/42636199-718b3d22-85e8-11e8-9403-99fe54eb7165.jpg)


That's the on you should put in the HTTP HEADERS pane bottom-left corner

**Comment by Muppet-training** [comment]
I don't event get to that part.. 
I just have a "message": "Your token is invalid..
I'm expecting a screen like yours.

Thanks so much for the help,
Does my screen shot help?
![capture2](https://user-images.githubusercontent.com/10666103/42680095-5a7b5dd0-86c7-11e8-963f-49bebfc80d77.JPG)

**Comment by guledali** [comment]
@Muppet-training Did you solve the problem

**Comment by timot7782** [comment]
make sure the service and the stage are the same as in the endpoint
for example, 
in prisma.yml,
```
service: **hacker-news-node**
stage: _dev_
```

in index.js,
```
const server = new GraphQLServer({
    typeDefs: './src/schema.graphql',
    resolvers,
    context: req => ({
        ...req,
        db: new Prisma({
            typeDefs: 'src/generated/prisma.graphql',
            endpoint: 'https://eu1.prisma.sh/myName01-05e8a6/**hacker-news-node**/_dev_',
            secret: 'secret123',
            debug: true,
        }),
    })
})
```

**Comment by guledali** [comment]
As my people pointed out you have to put the secret from prisma.yml in index.js

Like this 
![screenshot 71 _li](https://user-images.githubusercontent.com/20647282/42715611-0be65d74-86f8-11e8-9518-0ff3627bf080.jpg)
![screenshot 70 _li](https://user-images.githubusercontent.com/20647282/42715620-13efafe8-86f8-11e8-9d5c-bf16e48464a2.jpg)


Apparently this is an issue with how to they missed adding that information
![screenshot 72](https://user-images.githubusercontent.com/20647282/42715673-4ea7e588-86f8-11e8-9b23-b22fc5c395c8.png)

**Comment by timot7782** [comment]
yeah i was about to comment on how you didnt include the secret on the graphQLServer

**Comment by guledali** [comment]
@timot7782  I just made an issue on this #717 I hope they add this to the tutorial. I followed the tutorial strictly and they didn't include it.

**Comment by guledali** [comment]
There is already an issue on this #711

**Comment by marktani** [comment]
I merged PR #709, thanks for your feedback
